mesos-docker/run/run.sh

```diff
@@ -524,59 +544,59 @@ function show_help {
 
   Options:
 
+  --cpu-th the percentage of the host memory to use for slaves. Default: 0.5.
+  --hadoop-binary-file the hadoop binary file to use in docker configuration (optional, if not present tries to download the image).
   -h|--help prints this message.
-  -q|--quiet no output is shown to the console regarding execution status.
-  --number-of-slaves number of slave mesos containers to create (optional, defaults to 1).
-  --hadoop-binary-file  the hadoop binary file to use in docker configuration (optional, if not present tries to download the image).
-  --spark-binary-file  the hadoop binary file to use in docker configuration (optional, if not present tries to download the image).
   --image-version  the image version to use for the containers (optional, defaults to the latest hardcoded value).
-  --with-zk sets master as a zookeeper node.
-  --with-mararthon starts marathon node requires zookeeper.
+  --mem-th the percentage of the host cpus to use for slaves. Default: 0.5.
+  --mesos-master-config parameters passed to the mesos master (specific only and common with slave).
+  --mesos-slave-config parameters passed to the mesos slave(specific only an docmmon with the master).
   --no-hdfs to ignore hdfs installation step
-  --no-hdfs to ignore hdfs installation step.
   --no-shuffle-service to not start the external scheduler service.
+  --number-of-slaves number of slave mesos containers to create (optional, defaults to 1).
+  -q|--quiet no output is shown to the console regarding execution status.
+  --slaves-cfg-file provide a slave configuration file to pass specific attributes per slave
+  --spark-binary-file the hadoop binary file to use in docker configuration (optional, if not present tries to download the image).
   --update-image-mesos-at-version update the mesos on docker image with a specific version
     (use version strings from here: http://mesos.apache.org/downloads/ eg. 0.27.0). If version installed is the same nothing will be updated.
     Can be set with env var: MIT_DOCKER_MESOS_VERSION=0.27.0. Command line argument takes precedence.
-  --mem-th the percentage of the host cpus to use for slaves. Default: 0.5.
-  --cpu-th the percentage of the host memory to use for slaves. Default: 0.5.
-  --mesos-master-config parameters passed to the mesos master (specific only and common with slave).
-  --mesos-slave-config parameters passed to the mesos slave(specific only an docmmon with the master).
-  --slaves-cfg-file provide a slave configuration file to pass specific attributes per slave
+  --with-history-server starts history server on master container, it will use hdfs://hdfs_host:hdfs_port/spark_history folder for app event logging.
+  --with-mararthon starts marathon node requires zookeeper.
+  --with-zk sets master as a zookeeper node.
```

Comment:
How about "starts zookeeper on master container", I think it's clearer than sets master as a zookeeper node.